ui/widgets/datepicker.js

```diff
@@ -863,6 +865,10 @@ $.extend( Datepicker.prototype, {
 				origyearshtml = inst.yearshtml = null;
 			}, 0 );
 		}
+
+		if ( onUpdateDatepicker ) {
+			onUpdateDatepicker.apply( ( inst.input ? inst.input[ 0 ] : null ), [ inst ] );
```

Comment:
This would probably be slightly smaller:
```suggestion
			onUpdateDatepicker.call( ( inst.input ? inst.input[ 0 ] : null ), inst );
```